services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,39 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
-  return ERR_CODE_SUCCESS;
+  if (event == NULL)
+    return ERR_CODE_INVALID_QUEUE_MSG;
+  if (xQueueSend(thermalMgrQueueHandle, event, (TickType_t )0) == pdTRUE)
+    return ERR_CODE_SUCCESS;
+  else
+    return ERR_CODE_UNKNOWN;
```

Comment:
if `xQueueSend` fails its because the queue is full so that error code should be used instead